src/main/java/bowling/domain/Bowlings.java

```diff
@@ -0,0 +1,54 @@
+package bowling.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Bowlings {
+
+  private int current = 1;
+  private final List<Player> players;
+
+  public Bowlings(List<String> playerNames) {
+    players = new ArrayList<>();
+    for (String playerName : playerNames) {
+      players.add(Player.ofName(playerName));
+    }
```

Comment:
여기는 정말로 `Stream` 사용하시면 간략하게 될 것 같은데요?

```suggestion
    this.players = playerNames.stream()
                          .map(Player::ofName).collect(Collectors.toList());